frontend/public/redux.js

```diff
@@ -15,7 +15,7 @@ const reducers = combineReducers({
   [monitoringReducerName]: monitoringReducer,
 });
 
-const store = createStore(reducers, {}, applyMiddleware(thunk));
+const store = createStore(reducers, window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(), applyMiddleware(thunk));
```

Comment:
Not sure we want to check in this change

Reply:
Definitely not, but since its a WIP I've left it there... At least its a WIP indicator :smiley: